service/src/main/java/dk/kvalitetsit/stakit/service/model/SubscriptionModel.java

```diff
@@ -2,6 +2,12 @@
 
 import java.util.List;
 import java.util.UUID;
+import java.util.stream.Stream;
 
-public record SubscriptionModel(String email, List<UUID> groups, boolean announcements) {
+public record SubscriptionModel(UUID uuid, String email, List<UUID> groups, boolean announcements) {
+
+
+    public static SubscriptionModel createInstance(SubscriptionModel sub1, SubscriptionModel sub2) {
+        return new SubscriptionModel(sub1.uuid(), sub1.email(), Stream.concat(sub1.groups.stream(), sub2.groups.stream()).distinct().toList(), sub1.announcements());
```

Comment:
I en klasse/record har du noget funktionalitet til at lægge to lister sammen. Personligt synes jeg ikke det hører hjemme i en record. Du kan lave det forholdsvis elegant i dk.kvalitetsit.stakit.service.SubscriptionManagementServiceImpl#getSubscriptions med noget i stil med 
```
        response.forEach(s -> {
            subs.computeIfAbsent(s.subUuid(), (k) -> new SubscriptionModel(s.subUuid(), s.email(), new ArrayList<>(), s.announcements()))
                    .groups().add(s.groupUuid());
        });
```